src/biip/gs1/_messages.py

```diff
@@ -78,6 +80,31 @@ def parse(
 
         return cls(value=value, element_strings=element_strings)
 
+    @classmethod
+    def parse_hri(cls, value: str) -> GS1Message:
+        """Parse the GS1 string given in HRI (human readable interpretation) format.
+
+        Args:
+            value: The GS1 string to parse.
+
+        Returns:
+            A message object with one or more element strings.
+
+        Raises:
+            ParseError: If parsing of the data fails.
+        """
+        pattern = r"\((\d+)\)(\w+)"
+        matches = re.findall(pattern, value)
+        matches = [
+            (biip.gs1._application_identifiers._GS1_APPLICATION_IDENTIFIERS[ai], value)
```

Comment:
Because of the updated imports:

```suggestion
            (_GS1_APPLICATION_IDENTIFIERS[ai], value)
```